apps/mappings/imports/modules/base.py

```
@@ -205,7 +214,7 @@ def construct_payload_and_import_to_fyle(
         is_auto_sync_status_allowed = self.get_auto_sync_permission()
 
         filters = self.construct_attributes_filter(self.destination_field)
-
+        destination_attributes = DestinationAttribute.objects.filter(**filters)
```

Ruff (0.7.0)

Local variable `destination_attributes` is assigned to but never used

Remove assignment to unused variable `destination_attributes`

(F841)